pkg/operators/wasm/wasm.go

```diff
@@ -45,6 +46,8 @@ const (
 	// Current version of this API. This is used to check that the wasm module
 	// uses the same version.
 	apiVersion = 1
+
+	wasmInstanceVarName = "wasmInstance"
```

Comment:
If it is only for testing:
```suggestion
	wasmInstanceTestingVarName = "testingWasmInstance"
```